### PR TITLE
Several updates including team members and citations

### PR DESCRIPTION
**Pull request by jameseastwood**
The website has fallen out of date in the absence of a Project Manager. This PR will update the Team page to include several team members who did not yet appear. It will also update the How To Cite page with current citations, ensure that the How To Join page contains current information, and add a news item announcing the release of Sage 2.1.0.

### CODE REVIEW

**Comment by jameseastwood** [comment]
Interchange and Bespokefit still need release DOIs to add to the How-To-Cite page. I believe that @mattwthompson set up the necessary zenodo integration to generate these on release, but that mechanism has not been triggered yet.

**Comment by mattwthompson** [comment]
Since then I've made an Interchange release, so it's set up there: https://zenodo.org/record/8147765

BespokeFit won't have that until a release is made, and I don't think there's one planned in the short term

**Comment by jameseastwood** [comment]
Is it possible to 

1. Correct the typo in the title of Zenodo record and
2. add this record to the [Open Force Field Initiative community](https://zenodo.org/communities/openforcefield)?

**Comment by mattwthompson** [comment]
1. Done - I had to fake a new version by switching out the zip archive for a tarball, so there's a new DOI despite it being the same version (of the software). I think this should be okay, and I hope any issues should naturally correct themselves when the next release is made. Here it is: https://zenodo.org/record/8179774
2. Done - or at least I think it's in your (or Mobley's ?) boat to click something else somewhere: 
image

File: content/community/news/general/sage2.1.0-release/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+The easiest way to access the latest force field files is to install the latest version of the [openff-forcefields](https://github.com/openforcefield/openff-forcefields) package:
+`conda install --yes -c conda-forge "openff-forcefields >=2.1.0"`
+and load the force field via the toolkit:
+```python
+from openff.toolkit.typing.engines.smirnoff import ForceField
```
Comment (j-wags) [inline content/community/news/general/sage2.1.0-release/index.md]:
```suggestion
from openff.toolkit import ForceField
```

File: content/community/news/general/sage2.1.0-release/index.md~ [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+date: "2023-07-17T00:00:00+00:00"
```
Comment (j-wags) [inline content/community/news/general/sage2.1.0-release/index.md~]: This is an emacs autosave file - it should be removed.

File: content/science/how-to-cite/_index.md [hunk 3]
```diff
@@ -8,19 +8,28 @@ When using Open Force Field parameters or software in your research, please cite
 
 ### Force fields
 
-* Parsley ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Specific version DOI](https://github.com/openforcefield/openff-forcefields#open-force-fields))
+* Parsley ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Specific version DOI](https://zenodo.org/record/8048547))
```
Comment (j-wags) [inline content/science/how-to-cite/_index.md]: Kinda your call here, but I don't think this zenodo link is an improvement - It refers to a single version of the FF package, while each FF release gets a new version of the FF package (though each new package includes all the contents of the previous ones). In theory users should look at the table at the old link and cite the proper DOI corresponding to the FF version the used, but I don't think it really matters as long as the citations are caught somewhere.

File: content/science/how-to-cite/_index.md [hunk 4]
```diff
@@ -8,19 +8,28 @@ When using Open Force Field parameters or software in your research, please cite
...
 
-* Sage ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Specific version DOI](https://github.com/openforcefield/openff-forcefields#open-force-fields))
+* Sage ([Boothroyd et al., 2023](https://pubs.acs.org/doi/10.1021/acs.jctc.3c00039) + [Specific version DOI](https://zenodo.org/record/8048547))
  
 ### Software
 
-* OpenFF Toolkit ([Mobley et al., 2018](https://pubs.acs.org/doi/10.1021/acs.jctc.8b00640) + [Specific version DOI](https://zenodo.org/record/6310995#.YkdRtpPMLMI))
+* OpenFF Toolkit ([Mobley et al., 2018](https://pubs.acs.org/doi/10.1021/acs.jctc.8b00640) + [Specific version DOI](https://zenodo.org/record/8102071))
```
Comment (j-wags) [inline content/science/how-to-cite/_index.md]: This is another somewhat-arbitrary "latest on the day the website was updated" DOI. In this case we should use a perma-latest DOI like https://doi.org/10.5281/zenodo.597754 (you can find the perma-DOI links on the right side of any github-linked zenodo entry, below the version scrollbar)

File: content/science/how-to-cite/_index.md [hunk 5]
```diff
@@ -8,19 +8,28 @@ When using Open Force Field parameters or software in your research, please cite
...
 
-* OpenFF BespokeFit ([Zenodo DOI](https://zenodo.org/record/5172174#.YkdU1JPMLMI))
+* OpenFF BespokeFit ([Horton et al., 2022](https://pubs.acs.org/doi/full/10.1021/acs.jcim.2c01153))
```
Comment (j-wags) [inline content/science/how-to-cite/_index.md:19]: 👍

**Review by j-wags — APPROVED** [review]
Requested some small cleanups (will merge the ones I can fix myself). I provided some feedback for you to consider on citations, but you can merge it in any state you'd like.

File: content/community/news/general/sage2.1.0-release/index.md (hunk 1 above)
Comment (j-wags) [reply content/community/news/general/sage2.1.0-release/index.md]: We simplified the import pathways in version 0.11 of the OFF Toolkit, to great fanfare 🎉

File: content/community/news/general/sage2.1.0-release/index.md~ (hunk 2 above)
Comment (jameseastwood) [reply content/community/news/general/sage2.1.0-release/index.md~]: Aargh, I thought I had removed all of these. Thanks for catching it.

File: content/science/how-to-cite/_index.md (hunk 4 above)
Comment (jameseastwood) [reply content/science/how-to-cite/_index.md]: Good catch -- I was attempting to use the "all versions" doi and must have copied the wrong link.

**Pull request merged**